src/fiatconnect/kyc/KycPending.tsx

```diff
@@ -1,26 +1,106 @@
-import * as React from 'react'
-import { StyleSheet, Text } from 'react-native'
+import React, { useMemo } from 'react'
+import { StyleSheet, Text, View } from 'react-native'
 import { SafeAreaView } from 'react-native-safe-area-context'
 import { StackScreenProps } from '@react-navigation/stack'
 import { StackParamList } from 'src/navigator/types'
 import { Screens } from 'src/navigator/Screens'
+import { navigateHome } from 'src/navigator/NavigationService'
+import { useTranslation } from 'react-i18next'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { FiatExchangeEvents } from 'src/analytics/Events'
+import { KycStatus as FiatConnectKycStatus } from '@fiatconnect/fiatconnect-types'
+import fontStyles from 'src/styles/fonts'
+import colors from 'src/styles/colors'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import ClockIcon from 'src/icons/ClockIcon'
+import CircledIcon from 'src/icons/CircledIcon'
+import BankIcon from 'src/icons/BankIcon'
+import { useSelector } from 'react-redux'
+import { localCurrencyExchangeRatesSelector } from 'src/localCurrency/selectors'
+import getNavigationOptions from 'src/fiatconnect/kyc/getNavigationOptions'
 
 type Props = StackScreenProps<StackParamList, Screens.KycPending>
 
-// TODO implement designs:
-// https://www.figma.com/file/bwIY7peeykI8K4wTrXs4iN/Fiat-Connect?node-id=2885%3A40789
-function KycPending(_props: Props) {
+function KycPending({ route, navigation }: Props) {
+  const exchangeRates = useSelector(localCurrencyExchangeRatesSelector)
+  useMemo(() => {
+    navigation.setOptions(
+      getNavigationOptions({
+        fiatConnectKycStatus: FiatConnectKycStatus.KycPending,
+        quote: route.params.quote,
+        exchangeRates,
+      })
+    )
+  }, [exchangeRates])
```

Comment:
seems like if `route.params.quote` changed, we'd also want this to change. so maybe it should be added to deps. wdyt?

Reply:
idk how it could change, but even if it doesn't ever change, it still doesn't hurt to be there

Reply:
same with the other screens in this PR that have useMemo